src/simd_basic.h

```diff
@@ -37,13 +37,27 @@ namespace quadiron {
 namespace simd {
 
 template <typename T>
-inline VecType Card(T q)
+inline VecType Card(T q);
+template <>
+inline VecType Card<uint16_t>(uint16_t q)
```

Comment:
Don't use CamelCase for function name (should be `card`)

Reply:
- [x] rename